docs/api.rst

```diff
@@ -1166,6 +1166,45 @@ to handle it, which defaults to print a traceback and ignoring the exception.
     :param user: The user that interested.
     :type user: :class:`ScheduledEventUser`
 
+.. function:: on_automod_rule_create(rule)
```

Comment:
Should `rule` not be `action` here to remain consistent with your other documentation? Or the other way around